apps/epic-react/src/server/rewrite-next-response-to-path.ts

```diff
@@ -0,0 +1,7 @@
+import {NextRequest, NextResponse} from 'next/server'
+
+export function rewriteToPath(path: string, req: NextRequest) {
+  const baseUrlForRewrites = req.nextUrl.clone()
+  baseUrlForRewrites.pathname = path
+  return NextResponse.rewrite(baseUrlForRewrites)
```

Comment:
Probably we want to use `redirect()` method instead of `rewrite()` here? So that `/learn` get explicitly visible in address bar?